Author: Francois Marier <francois@debian.org>
Description: don't strip the main binary, it should be done in debian/rules (#436626)
--- a/Makefile
+++ b/Makefile
@@ -41,36 +41,28 @@ sense: chklastlog chkwtmp ifpromisc chkp
 
 chklastlog:   chklastlog.c
 	${CC} ${CFLAGS} -o $@ chklastlog.c
-	@strip $@
 
 chkwtmp:   chkwtmp.c
 	${CC} ${CFLAGS} -o $@ chkwtmp.c
-	@strip $@
 
 ifpromisc:   ifpromisc.c
 	${CC} ${CFLAGS} ${LDFLAGS}  -D_FILE_OFFSET_BITS=64 -o $@ ifpromisc.c
-	@strip $@
 
 chkproc:   chkproc.c
 	${CC} ${LDFLAGS} -o $@ chkproc.c
-	@strip $@
 
 chkdirs:   chkdirs.c
 	${CC} ${LDFLAGS} -o $@ chkdirs.c
-	@strip $@
 
 check_wtmpx:   check_wtmpx.c
 	${CC} ${LDFLAGS} -o $@ check_wtmpx.c
-	@strip $@
 
 chkutmp:   chkutmp.c
 	${CC} ${LDFLAGS} -o $@ chkutmp.c
-	@strip $@
 
 
 strings-static:   strings.c
 	${CC} ${STATIC} ${LDFLAGS} -o $@ strings.c
-	@strip $@
 
 clean:
 	rm -f ${OBJS} core chklastlog chkwtmp ifpromisc chkproc chkdirs check_wtmpx strings-static chkutmp
